l10n_br_base/models/res_partner_bank.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2009 Gabriel C. Stabel
+# Copyright (C) 2009 Renato Lima (Akretion)
+# Copyright (C) 2012 Raphaël Valyi (Akretion)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import fields, models
+
+
+class ResPartnerBank(models.Model):
```

Comment:
I don't see reason to change file formation, should just change import order:

https://github.com/OCA/l10n-brazil/blob/12.0/l10n_br_base/models/res_partner_bank.py